### PR TITLE
Fix: move svg icons to tsx

### PR DESCRIPTION
**Pull request by akushniruk**

### CODE REVIEW

File: src/assets/images/ChevronIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+interface ChevronIconProps {
+    className?: string;
+}
+
+export const ChevronIcon: React.FC<ChevronIconProps> = (props: ChevronIconProps) => (
+    <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (akhlopiachyi) [inline src/assets/images/ChevronIcon.tsx]: Use className property here

File: src/assets/images/LockedIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+interface LockedIconProps {
+    className?: string;
+}
+
+export const LockedIcon: React.FC<LockedIconProps> = (props: LockedIconProps) => (
+    <svg width="328" height="400" viewBox="0 0 328 400" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (akhlopiachyi) [inline src/assets/images/LockedIcon.tsx]: Use className property here

File: src/containers/Confirm/Documents/index.tsx [hunk 3]
```diff
@@ -230,7 +229,7 @@ class DocumentsComponent extends React.Component<Props, DocumentsState> {
                 onClick={this.handleFileDelete(index)}
             >
                 {scan.name.slice(0, 27)}...&nbsp;
-                <img alt="close" src={close}/>
+                    <CloseIcon className="close-icon" />
```
Comment (akhlopiachyi) [inline src/containers/Confirm/Documents/index.tsx]: fix indentation

File: src/containers/Header/index.tsx [hunk 4]
```diff
@@ -44,7 +42,7 @@ type Props = ReduxProps & HistoryProps & DispatchProps & InjectedIntlProps;
 // tslint:disable jsx-no-multiline-js
```
Comment (akhlopiachyi) [inline src/containers/Header/index.tsx]: remove comment

File: src/containers/Sidebar/index.tsx [hunk 5]
```diff
@@ -51,23 +54,21 @@ class SidebarContainer extends React.Component<Props, State> {
...
-        const lightBox = isLight ? 'light-box' : '';
         const languageName = lang.toUpperCase();
 
         const languageClassName = classnames('dropdown-menu-language-field', {
             'dropdown-menu-language-field-active': isOpenLanguage,
         });
 
         return (
-            <div className={`pg-sidebar-wrapper ${lightBox} pg-sidebar-wrapper--${isActive ? 'active' : 'hidden'}`}>
+            <div className={`pg-sidebar-wrapper pg-sidebar-wrapper--${isActive ? 'active' : 'hidden'}`}>
```
Comment (akhlopiachyi) [inline src/containers/Sidebar/index.tsx]: Please, use classnames

**Comment by pkucherenk0** [comment]
1. The cancel order icon has no cursor pointer on the Trading page -> Open orders, Same in Order page -> Open and All tabs.
Selection_752
2. The API keys delete icons actually not an icon, we should replace it with an icon
Selection_753
3. Aline disable 2FA close icon
Selection_754

**Comment by akushniruk** [comment]
done

File: src/assets/images/background.svg [hunk 6]
```diff
@@ -1,3 +1,3 @@
...
-</svg>
+</svg>
```
[the remark this answers is not in the document]
Comment (akushniruk) [reply src/assets/images/background.svg:3]: done

**Pull request merged**